app/server-utils/stampy.ts

```diff
@@ -211,21 +211,25 @@ const head = (item: any) => {
 }
 const extractText = (markdown: string) => head(markdown)?.replace(/^```|```$/g, '')
 const extractLink = (markdown: string) => markdown?.replace(/^.*\(|\)/g, '')
-const convertToQuestion = (title: string, v: CodaRow['values']): Question => ({
-  title,
-  pageid: extractText(v['UI ID']),
-  text: v['Rich Text'] ? urlToIframe(md.render(extractText(v['Rich Text']))) : null,
-  answerEditLink: extractLink(v['Edit Answer']).replace(/\?.*$/, ''),
-  tags: ((v['Tags'] || []) as Entity[]).map((e) => e.name),
-  relatedQuestions:
-    v['Related Answers'] && v['Related IDs']
-      ? v['Related Answers'].map(({name}, i) => ({
-          title: name,
-          pageid: extractText(v['Related IDs'][i]),
-        }))
-      : [],
-  status: v['Status']?.name as QuestionStatus,
-})
+const convertToQuestion = (title: string, v: CodaRow['values']): Question => {
+  const renderedText = renderText(v['Rich Text'])
+  const text = renderedText ? urlToIframe(renderedText) : renderedText
```

Comment:
Could you move the `urlToIframe` into `renderText`, please?

Reply:
Yes, great idea, I will do that 💡

Reply:
Implemented in https://github.com/StampyAI/stampy-ui/commit/a679fb17af839571520a946d3486f35aae6d5ca2